interoperator/pkg/webhook/add_default_server.go

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright 2018 The Service Fabrik Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package webhook
+
+import (
+	server "github.com/cloudfoundry-incubator/service-fabrik-broker/interoperator/pkg/webhook/default_server"
```

Comment:
`default_server` name does not sound good. Can we change it ?

Reply:
Sure, all the names which have been marked for review here and below were generated by kubebuilder. I'll be discussing better names IRL before making the updates.